.claude/commands/xata-read.yaml

```
@@ -0,0 +1,174 @@
+  steps:
+    - Parse command arguments
+    - Import xata client
+    - Build query based on parameters
+    - Handle id vs filter queries
+    - Apply columns, limit, offset, sort
+    - Execute query and format results
+    - Return structured response
+  
+  code_template: |
+    import { xata } from '@packages/db/src/xata-typescript-sdk/client';
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)